bundles/org.openhab.ui.dashboard/ESH-INF/i18n/dashboard_nl.properties

```diff
@@ -1,28 +1,6 @@
-index.welcome = Welkom bij openHAB 2
-index.location-info = Als u uw browser toegang geeft tot uw locatie, zal openHAB deze gebruiken voor weer- en astro-informatie.
-index.your-location = Je locatie:
+#X-Generator: crowdin.com
```

Comment:
Hey this is a pitty

Reply:
Yes, I just got to that :)

Reply:
Just to be clear: Please join https://crowdin.com/project/openhab-core 😉